index.html

```diff
@@ -1,62 +1,94 @@
-<!DOCTYPE html>
 <html>
 
 <head>
-    <title>Worlds Best Calculator</title>
-
-    <!-- CSS only -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-
     <style>
+        * {
```

Comment:
Nice use of the `*` selector here.  Other css looks good but is missing the `px` with the numbers.

For example see the use of px with numbers in this snippet.
```
.button {
            width: 50px;
            height: 50px;
            font-size: 30px;
            margin: 5px;
            cursor: pointer;
        }
```

See this page for more help https://www.w3schools.com/cssref/css_units.asp